app/services/current-device.js

```diff
@@ -22,4 +22,43 @@ export default class CurrentDeviceService extends Service {
 
     return device;
   }
+
+  async configure(ssid, password) {
+    const response = await fetch(
+      `${ENV.APP.BASE_API_URL}/api/v1/device/${this.data.serialNumber}/configure`,
+      {
+        method: "POST",
+        body: JSON.stringify({
+          configuration: {
+            ssid,
+            password,
+          },
+        }),
+      }
+    );
```

Comment:
It could get cleanup up a bit if http service was used
```suggestion
    const response = await this.http.post(
      `/api/v1/device/${this.data.serialNumber}/configure`,
      JSON.stringify({
          configuration: {
            ssid,
            password,
          }),
      }
    );
```